pkg/tcpip/stack/stack.go

```diff
@@ -46,6 +47,29 @@ const (
 	resolutionAttempts = 3
 )
 
+// Packet forwarding flags. Forwarding settings for different network protocols
+// are stored as bit flags in an uint32 number.
+const (
+	forwardingIPv4 uint32 = 1 << iota
+
+	// forwardingFake is used to test package forwarding with a fake protocol.
+	forwardingFake
+)
+
+func getForwardingFlag(protocol tcpip.NetworkProtocolNumber) uint32 {
+	var flag uint32
+	switch protocol {
+	case header.IPv4ProtocolNumber:
+		flag = forwardingIPv4
```

Comment:
Would it be possible to add an option for IPv6? Netstack supports IPv6 and supported IPv6 forwarding prior to this PR. No need to plumb it all the way to a proc file though.

Reply:
IP forwarding in master branch is always off. There is no way to change ipv6 forwarding

Reply:
comment is added